sos/report/plugins/networking.py

```diff
@@ -27,7 +27,8 @@ class Networking(Plugin):
         ("namespaces", "Number of namespaces to collect, 0 for unlimited. " +
          "Incompatible with the namespace_pattern plugin option", "slow", 0),
         ("ethtool_namespaces", "Define if ethtool commands should be " +
-         "collected for namespaces", "slow", True)
+         "collected for namespaces", "slow", True),
+        ("eepromdump", "collect 'ethtool -e' for all devices", "fast", False)
```

Comment:
We don't really do anything with the "fast"/"slow" metadata but since the plugin is already using the "slow" string for other options (and since this collection is known to cause "hiccups" of various kinds), perhaps we should use it for this new option?